server/views/eligibility-checker/contain-elephant-ivory.html

```diff
@@ -34,7 +34,7 @@
                   <li>fake ivory, such as ivorine</li>
                   <li>certain types of animal bone</li>
                 </ul>
-                <p>If you are not sure about the item you want to sell or buy, you should consult an expert first, such as a registered antiques dealer or auctioneer who specialises in ivory.</p>'
+                <p>If you are not sure about the item you want to sell or buy, you should consult an expert first, such as an antiques dealer or auctioneer who specialises in ivory.</p>'
```

Comment:
Add IDs to the 2 P tags and the UL tag in this HTML block and add unit tests to check for existence and value of those elements.